ports/samd/variants/samd21/mpconfigmcu.h

```diff
@@ -0,0 +1,36 @@
+// Deinitions common to all SAMD21 boards
+#include "samd21.h"
+
+#define MICROPY_CONFIG_ROM_LEVEL        (MICROPY_CONFIG_ROM_LEVEL_CORE_FEATURES)
+
+// MicroPython emitters
+#define MICROPY_EMIT_THUMB              (0)
+#define MICROPY_EMIT_INLINE_THUMB       (0)
+#define MICROPY_MODULE_BUILTIN_INIT     (1)
+
+#define MICROPY_FLOAT_IMPL              (MICROPY_FLOAT_IMPL_FLOAT)
+#define MICROPY_PY_BUILTINS_COMPLEX     (0)
+#define MICROPY_PY_MATH                 (0)
+#define MICROPY_PY_CMATH                (0)
+
+#define VFS_BLOCK_SIZE_BYTES            (1536) // 24x 64B flash pages;
+
+#define MICROPY_HW_UART_TXBUF           (1)
+
+#define CPU_FREQ                        (48000000)
+#define DFLL48M_FREQ                    (48000000)
+
+// Use VfsLfs's types for fileio/textio
+#define mp_type_fileio mp_type_vfs_lfs1_fileio
+#define mp_type_textio mp_type_vfs_lfs1_textio
```

Comment:
These two defines are no longer needed, they can be removed.

Reply:
OK